error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -937,7 +937,10 @@ Flux<T> after(Mono<T> mono) {
 
   /** Prefer direct invocation of {@link Mono#then()}} over more contrived alternatives. */
   static final class MonoThen<T> {
+    // XXX: Drop the warning suppression annotation once NullAway in JSpecify mode understands that
+    // `Mono<Void>` is a subtype of `Mono<@Nullable Void>`.
     @BeforeTemplate
+    @SuppressWarnings("NullAway")
     Mono<@Nullable Void> before(Mono<T> mono) {
```

Comment:
There are many similar expressions. NullAway can be appeased by simply replacing `Mono<@Nullable Void>` with `Mono<Void>`, but then [VoidMissingNullable](https://errorprone.info/bugpattern/VoidMissingNullable) complains.

Reply:
Please file an issue on this one.  We have hacks for `@Nullable Void` but we should disable them in JSpecify mode.

Reply:
Check! I filed uber/NullAway#1172.